src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -170,7 +170,7 @@
     
     private static final byte[] emptyUserData = new byte[0];
     
-    private static final boolean debug = false;
+    private static final boolean debug = PropertiesHelper.systemHelper.getBoolean("com.ms.silverking.cloud.dht.daemon.storage.NamespaceStorage.debug", false);;
```

Comment:
Should use a property name for this instead of the hardcode. Something like:
private static final String    debugProperty = NamespaceStore.class.getCanonicalName() +".Debug";